src/app/ubs-user/ubs-user-orders-list/ubs-user-orders-list.component.ts

```diff
@@ -51,4 +55,16 @@ export class UbsUserOrdersListComponent {
       }
     });
   }
+
+  public sortingOrdersByData(): void {
+    this.orders.sort((a: IOrderInfo, b: IOrderInfo): number => {
+      if (a.generalOrderInfo.dateFormed > b.generalOrderInfo.dateFormed) {
+        return -1;
+      }
+      if (a.generalOrderInfo.dateFormed < b.generalOrderInfo.dateFormed) {
+        return 1;
+      }
+      return 0;
+    });
+  }
```

Comment:
You sort object by only one property, so
return a.generalOrderInfo.dateFormed < b.generalOrderInfo.dateFormed ? 1 : -1;